app/models/sample.rb

```diff
@@ -79,8 +85,14 @@ def sample_postprocess_s3_path
     "s3://#{SAMPLES_BUCKET_NAME}/#{sample_path}/postprocess"
   end
 
-  def sorted_taxid_annotated_fasta_s3_path
-    "#{sample_postprocess_s3_path}/#{SORTED_TAXID_ANNOTATED_FASTA}"
+  def s3_paths_for_taxon_byteranges
+    # by tax_level and hit_type
+    { 1 => { 'NT' => "#{sample_postprocess_s3_path}/#{SORTED_TAXID_ANNOTATED_FASTA}",
```

Comment:
instead of 1, 2, 3 use TaxonCount::TAX_LEVEL_SPECIES, etc

Reply:
ok